pkgs/by-name/mi/mimikatz/package.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, fetchzip
+, stdenvNoCC
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "mimikatz";
+  version = "2.2.0-20220919";
+
+  src = fetchzip {
+    url = "https://github.com/gentilkiwi/mimikatz/releases/download/${finalAttrs.version}/mimikatz_trunk.zip";
+    hash = "sha256-wmatI/rEMziBdNiA3HE3MJ0ckdpvsD+LdbB7SKOYdI0=";
+    stripRoot=false;
+  };
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/mimikatz
+    cp -a * $out/share/mimikatz
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "https://github.com/gentilkiwi/mimikatz";
+    description = "A little tool to play with Windows security";
+    mainProgram = "mimikatz";
+    license = with lib.licenses; [ cc-by-40 ];
+    maintainers = with lib.maintainers; [ d3vil0p3r ];
+    platforms = lib.platforms.all;
```

```suggestion
    platforms = lib.platforms.windows;
```

Apparently won't work under Linux or macOS.

---

I prefer to leave as `all` because it is used for pentesting, so, if I have a pentesting OS Nix-based, I need to have these .exe files to infiltrate inside a Windows target machine.

---

Mimikatz should be used only by InfoSec professional, so, what I suggest is to remove `mainProgram` because no useful in this context, but I would still keep it as `all` as explained above. Look also `powersploit` package that has the same principle.

---

Alright, let's drop `mainProgram` then.